apps/gallery/index.html

```diff
@@ -20,6 +20,7 @@
     <meta name="availableLanguages" content="ar, en-US, fr, zh-TW">
     <link rel="localization" href="locales/gallery.{locale}.properties">
     <link rel="localization" href="/shared/locales/date/date.{locale}.properties">
+    <link rel="localization" href="/shared/locales/media/media.{locale}.properties">
```

Comment:
It seems like a good idea to make these shared since we've got very similar audio controls in a bunch of places. And different video controls in the video app.  I wonder if calling it media/controls.{locale}.properties of playback.{locale}.properties would be better, though, so it could be focused just on the controls labels.